tensorflow/core/kernels/mkl/mkl_conv_ops.cc

```diff
@@ -348,18 +376,27 @@ class MklConvFwdPrimitive : public MklPrimitive {
           float op_scale = post_op_param.param[0];
           float op_alpha = post_op_param.param[1];
           float op_beta = post_op_param.param[2];
-          post_ops.append_eltwise(op_scale, post_op_param.alg, op_alpha,
+          // TODO(intel-tf): Enable this for int8 when using oneDNN v3.x
+          post_ops.APPEND_ELTWISE(op_scale, post_op_param.alg, op_alpha,
                                   op_beta);
         } else if (post_op_param.name == "sum") {
           DCHECK_EQ(post_op_param.param.size(), 1);
           float op_scale = post_op_param.param[0];
           post_ops.append_sum(op_scale);
         } else if (post_op_param.name == "output_scale") {
+#ifndef ENABLE_ONEDNN_V3
           if (post_op_param.param.size() == 1) {
             post_ops_attr.set_output_scales(0, post_op_param.param);
           } else {
             post_ops_attr.set_output_scales(2, post_op_param.param);
           }
+#else
+          // TODO(intel-tf): Enable this for int8 when using oneDNN v3.x
+          assert(post_op_param.param.size() == 1);
```

Comment:
We try to avoid `CHECK/DCHECK/assert` in new code. Ideally, we should make this function return a status. But since we are close to TF 2.13 branch cut, I think we can use `DCHECK` for now. Could you please change `assert` to `DCHECK` and add a TODO to make this function return an error status instead?

Reply:
Done, thanks.